### PR TITLE
Defaults: added mysql reconnect to ini defaults

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: container/root/etc/php5/mods-available/defaults.ini [hunk 1]
```diff
@@ -3,3 +3,4 @@ display_errors = Off
 html_errors = Off
 default_charset = 'UTF-8'
 date.timezone = 'UTC'
+mysqli.reconnect = 1
```
Comment (ko-be) [inline container/root/etc/php5/mods-available/defaults.ini:6]: Shouldn't this be in mysqli.ini or does not not matter @bryanlatten ?

Comment (chrishenry) [inline container/root/etc/php5/mods-available/defaults.ini:6]: Yea, or under the `[MySQLi]` header?

Comment (chrishenry) [inline container/root/etc/php5/mods-available/defaults.ini:6]: Actually, looks like that should work

**Comment by chrishenry** [comment]
:+1:

File: container/root/etc/php5/mods-available/defaults.ini (hunk 1 above)
Comment (bryanlatten) [inline container/root/etc/php5/mods-available/defaults.ini:6]: doesn't matter @ko-be, but this allows us to overlay the value more easily

**Comment by ko-be** [comment]
:+1:

**Pull request merged**